components/form-builder/panel/ElementPanel.tsx

```diff
@@ -466,29 +470,58 @@ const ElementPanelDiv = styled.div`
 export const ElementPanel = () => {
   const {
     form: { elements },
-    add,
   } = useTemplateStore();
 
-  if (!elements.length) {
-    return (
-      <button
-        style={{ marginBottom: 20 }}
-        className="gc-button gc-button--secondary"
-        onClick={() => {
-          add();
-        }}
-      >
-        Add form element
-      </button>
-    );
-  }
+  const introTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "Add an introduction" }],
+    },
+  ];
+
+  const confirmTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "Thank you for participating in the ice cream survey!" }],
+    },
+  ];
+
+  const policyTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "All email addresses will be not be shared." }],
```

Comment:
Too many "be"s 🐝

Reply:
Updated https://github.com/cds-snc/platform-forms-client/pull/1056/commits/68a8ad92b0e3e401a9724da30125141498f1defc